sdk/cognitivelanguage/ai-language-text/test/public/utils/customTestHelpter.ts

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  TextAnalysisAuthoringDeployProjectParameters,
+  TextAnalysisAuthoringTrainBodyParam,
+  getLongRunningPoller,
+  CreateProjectOptions,
+  TextAuthoringClient,
+  ExportedCustomSingleLabelClassificationProjectAssets,
+  ExportedCustomMultiLabelClassificationProjectAssets,
+  ExportedCustomEntityRecognitionProjectAssets,
+} from "@azure/ai-language-textauthoring";
+import { BlobServiceClient, ContainerClient, RestError } from "@azure/storage-blob";
+import { DefaultAzureCredential } from "@azure/identity";
+import path from "path";
+import decompress from "decompress";
+import {
+  customEntityAssets,
+  customMultiLabelAssets,
+  customSingleLabelAssets,
+} from "../customTestsAssets";
+import { assertEnvironmentVariable } from "@azure-tools/test-recorder";
+import { logger } from "../../../src/logger";
+import { readdir, rm } from "fs/promises";
+
+let files: string;
+const storageInputContainerName = "documents";
+const language = "en-us";
+const trainingConfigVersion = "2022-05-01";
+async function createStorageBlob(): Promise<ContainerClient> {
+  /* Return a container client */
+  const defaultAzureCredential = new DefaultAzureCredential();
+  // Get documents containers
+  const blobServiceClient = new BlobServiceClient(
+    assertEnvironmentVariable("STORAGE_ENDPOINT") || "string",
+    defaultAzureCredential
+  );
+  return blobServiceClient.getContainerClient(storageInputContainerName);
+}
+
+async function uploadDocuments(fileName: string): Promise<void> {
+  // Unpack zip files and upload to blob
+  const pathName = path.join(
+    __dirname.split("ai-language-text")[0],
+    "ai-language-text",
+    "test",
+    "public",
+    "utils"
+  );
+  await decompress(path.join(pathName, `${fileName}.zip`), pathName);
+  const dir = path.join(pathName, fileName);
+  const fileNames = await readdir(dir);
+
+  const client = await createStorageBlob();
+  for (const file of fileNames) {
+    if (file.endsWith(".txt")) {
+      const blob = client.getBlockBlobClient(file);
+      const filePath = path.join(dir, file);
+      await blob.uploadFile(filePath);
+    }
+  }
+}
+
+async function polling(client: TextAuthoringClient, project: any) {
+  try {
+    const poller = getLongRunningPoller(client, project);
+    await poller.pollUntilDone();
+  } catch (error) {
+    logger.info("Error encountered", error);
+    throw new RestError(`Poller encountered error here ${error}`);
+  }
```

Comment:
The poller for RLCs shouldn't throw errors, and even if it did, we can just let it bubble up:
```suggestion
  const poller = getLongRunningPoller(client, project);
  await poller.pollUntilDone();
```